spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpConnectionInterceptorSupport.java

```diff
@@ -249,8 +250,8 @@ public void removeDeadConnection(TcpConnection connection) {
 				return;
 			}
 			this.removed = true;
-			if (this.theConnection instanceof TcpConnectionInterceptorSupport && !this.theConnection.equals(this)) {
-				((TcpConnectionInterceptorSupport) this.theConnection).removeDeadConnection(this);
+			if (this.theConnection instanceof TcpConnectionInterceptorSupport tcpConnectionInterceptorSupport && !this.theConnection.equals(this)) {
```

Comment:
I probably will accept this PR as is.
Due to its size and simplicity, but for the future contributions, please, consider to make code lines not more than `120` symbols.